views/index.erb

```diff
@@ -47,13 +47,79 @@
 
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
-  console.log("I am Javascript!");
+	console.log("I am Javascript!");
 
-  var profile_values = document.getElementsByClassName('profile-value');
-  var total_words = 0;
+	var profile_values = document.getElementsByClassName('profile-value');
+	var total_words = 0;
+	var word_list = "";
+	for (var i in profile_values) {
+		if (profile_values[i].textContent != undefined) {
+			word_list += (profile_values[i].textContent + " ");
+		}
+	}
+	var filtered_list = word_list.split(" ").filter(function(val) { return val !== ""; });
+	total_words += filtered_list.length;
+	
 
   // TODO - Count the number of words in the profile.
   // hint - look at what the innerHTML property provides.
 
-  document.getElementById('word-count').innerHTML = total_words;
-</script>
\ No newline at end of file
+	document.getElementById('word-count').innerHTML = total_words.toString();
+	
+	
+	/* Responsive Layout */
+	var colRight = document.getElementsByClassName("col-right");
+	var colLeft = document.getElementsByClassName("col-left");
+	var wrap = document.getElementsByClassName("wrapper");
+
+	var smallWindow = function() {
+			for (var i=0; i < colRight.length; i++){
+				colRight.innerHTML += colRight[i].style.float="left";
+				colRight.innerHTML += colRight[i].style.width="100%";
+			}
+			for (var i=0; i < colLeft.length; i++){
+				colLeft.innerHTML += colLeft[i].style.width="100%";
+			}
+			for (var i=0; i < wrap.length; i++){
+				wrap.innerHTML += wrap[i].style.width="60%";
+			}
+	}
+
+	var largeWindow = function(){
+		for (var i=0; i < colRight.length; i++){
+			colRight.innerHTML += colRight[i].style.float="right";
+			colRight.innerHTML += colRight[i].style.width="45%";
+		}
+		for (var i=0; i < colLeft.length; i++){
+			colLeft.innerHTML += colLeft[i].style.width="50%";
+		}
+		for (var i=0; i < wrap.length; i++){
+			wrap.innerHTML += wrap[i].style.width="55%";
+		}
+	}
+	window.onload = function(){
+		if (window.innerWidth <= 755){
+			smallWindow();
+		}else{
+			largeWindow();
+		}
+	};
+	window.addEventListener('resize', function(){
+		if (window.innerWidth <= 755){
+			smallWindow();
+		}else{
+			largeWindow();
+		}
+	}, true);
```

Comment:
We'll show you a better way of doing in CSS, rather than JS. CSS has media queries that do this sort of thing whilst being more browser compatible, and quicker than the JS runtime